aspnetcore/blazor/forms/binding.md

```diff
@@ -154,17 +154,14 @@ Use the `FormName` parameter to assign a form name. Form names must be unique to
 
 Supplying a form name:
 
-* Is required on all forms that are submitted via interactivity with server rendering.
-* Not required for interactive rendering, which includes forms in Blazor WebAssembly apps and components marked with an interactive render mode.
+* Is required for all forms that are submitted by statically-rendered server-side components.
+* Isn't strictly required for forms that are submitted by interactively-rendered components, which includes forms in Blazor WebAssembly apps and components marked with an interactive render mode. However, we recommend supplying a unique form name for every form to prevent runtime form posting errors.
```

Comment:
Just to clarify, the `FormName` parameter isn't used at all during interactive rendering. I like recommendation for always supplying a form name, but specifically if it's possible for the component to be rendered statically (i.e., if you have a completely interactive app, then it's not necessary to specify a `FormName`).

Reply:
Yes, I was told by Steve or Jaiver to always *recommend* it the docs just in case. This is why I was angling this with "isn't strictly required." I think I'll remove the word "strictly" tho and leave the rest alone.

Reply:
I modified it a bit further as to why it's being recommended ...

> Isn't required for forms that are submitted by interactively-rendered components, which includes forms in Blazor WebAssembly apps and components marked with an interactive render mode. However, we recommend supplying a unique form name for every form to prevent runtime form posting errors if interactivity is ever dropped for a form.